docs/tutorials/intro_tutorial.ipynb

```diff
@@ -11,8 +11,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "**Important:** \n",
+    "- If you are just exploring Mesa and want the fastest way to execute the code we recommend executing this tutorial online in a Colab notebook. [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/projectmesa/mesa/blob/main/docs/tutorials/intro_tutorial.ipynb)\n",
```

Comment:
I didn't know that the intro tutorial is already readily executable.

Reply:
Aww crap... it is you just have to have  cell to do pip install mesa, but I did it in colab but not my local version, I will fix